src/style.css

```diff
@@ -102,11 +103,81 @@ button {
   border: 1px solid #888;
   padding: 0.5rem 0.9rem;
   border-radius: 4px;
+  position: relative;
+}
+
+.cart-badge {
+  position: absolute;
+  top: -9px;
+  right: -14px;
+  transform: translateY(-4%);
+  background-color: #1877f2;
+  color: white;
+  font-size: 0.8rem;
+  font-weight: bold;
+  padding: 3px 9px;
+  border-radius: 2px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
 }
 
 @media (min-width: 768px) {
   .button-container {
-    display: block;
+    display: flex;
+  }
+
+  .sign-in-btn {
+    background-color: #cecec9;
+    color: #000000;
+    font-size: 1rem;
+    font-weight: bold;
+    padding: 0.2rem 0.2rem;
+    border-radius: 5px;
+    cursor: pointer;
+    transition: all 0.3s ease;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  }
+
+  .sign-in-btn:hover {
+    background-color: #000000;
+    color: #f0f0f0;
```

Comment:
don't repeat the same css in all button. put all the general css in one class and use sperate class for difference in css